addons/mail/static/src/models/thread/thread.js

```diff
@@ -1995,6 +1996,12 @@ registerModel({
         guestMembers: many('Guest', {
             sort: '_sortGuestMembers',
         }),
+        /**
+         * States whether the current user has read access for this thread.
+         */
+        hasAccess: attr({
```

Comment:
Wondering about final field name cc @alexkuhn

Reply:
Looks more appropriate as `hasReadAccess`, just so that it can be `true/false` regardless of whether `hasWriteAccess` is `true/false`.

Just with `hasAccess`, it would imply it's more general, therefore `hasWriteAccess` being `false` would necessarily imply `hasAccess` being `false`, which is incorrect at a glance.

Reply:
@seb-odoo , Done all the changes. Thanks